File: android-native/FretboardHeatmap/app/src/main/java/com/example/fretboardheatmap/GuitarSpecs.kt
package com.example.fretboardheatmap

import androidx.compose.ui.unit.dp


/**
 * Provides data for guitar frets, strings, and root notes.
 */
object GuitarSpecs {
    // guitar frets
    val frets = listOf(90.dp, 85.dp, 80.dp, 75.dp, 71.dp, 67.dp, 63.dp, 60.dp, 56.dp, 53.dp, 50.dp, 47.dp)

    // guitar strings
    val strings = listOf(0.8.dp, 1.2.dp, 1.8.dp, 2.5.dp, 3.2.dp, 4.0.dp)

    // root notes
    val roots = listOf("G", "D", "C", "E", "A")
}

File: ios-native/FretboardHeatmap/GuitarSpecs.swift
//
//  GuitarSpecs.swift
//  FretboardHeatmap
//
//  Created by Sean Avery Suguitan on 2026-01-25.
//

import Foundation

///  Provides data for guitar frets, strings, and root notes
struct GuitarSpecs {
    // guitar frets
    static let frets: [CGFloat] = [90, 85, 80, 75, 71, 67, 63, 60, 56, 53, 50, 47]
    
    // guitar strings
    static let strings: [CGFloat] = [0.8, 1.2, 1.8, 2.5, 3.2, 4.0]
    
    // root notes
    static let roots = ["G", "D", "C", "E", "A"]
}

